scripts/clear-all-fault-leds.sh

```diff
@@ -68,7 +68,7 @@ then
         GetSubTreePaths sias "/xyz/openbmc_project/inventory" 0 1 "xyz.openbmc_project.State.Decorator.OperationalStatus" \
         | sed  's/ /\n/g' | tail -n+3 | awk -F "\"" '{print $2}' | while read -r line
     do
-        echo "$line" | grep "unit\|core" >/dev/null
+        echo "$line" | grep "powersupply\|dimm\|cpu\|fan" >/dev/null
```

Comment:
use dcm instead of cpu

Reply:
Then it can be different for different systems, like scm, so may not work for them.